raddb/radiusd.conf.in

```diff
@@ -286,6 +286,11 @@ log {
 	#
 #	timestamp = no
 
+	#
+	#  use_utc:: Enable to use UTC time.
```

Comment:
Should be "Print all timestamps in log messages as UTC"

Reply:
@alandekok if you get a moment, sure would be nice to make this a global toggle for printing value boxes too.  See telkom defect list :(